src/main/java/com/laytonsmith/core/functions/Minecraft.java

```diff
@@ -922,6 +929,23 @@ public Construct exec(Target t, Environment env, Construct... args) throws Cance
 				//Server Ip
 				retVals.add(new CString(server.getIp(), t));
 			}
+			if (index == 13 || index == -1) {
+				//Uptime
+				double uptime = (double)(new GregorianCalendar().getTimeInMillis() - ManagementFactory.getRuntimeMXBean().getStartTime());
```

Comment:
Is there a reason System.currentTimeMillis() won't work here instead of instantiating a new object?